guides/common/modules/ref_storage-requirements.adoc

```diff
@@ -13,66 +15,31 @@ ifdef::katello,satellite[]
 The runtime size was measured with {RHEL} 6, 7, and 8 repositories synchronized.
 endif::[]
 
-ifdef::foreman-el,katello,satellite[]
 == [[storage-el-8]]{EL} 8
 
-.Storage Requirements for a {ProjectServer} Installation
+.Storage Requirements for {SmartProxyServer} Installation
 [cols="1,1,1",options="header"]
 |====
 |Directory |Installation Size |Runtime Size
-
-|/var/log/ |10 MB |10 GB
-
-|/var/lib/pgsql |100 MB |20 GB
-
-|/usr | 3 GB | Not Applicable
-
-|/opt/puppetlabs | 500 MB | Not Applicable
-
 ifdef::katello,satellite,orcharhino[]
-|/var/lib/pulp/ |1 MB |300 GB
-
-|/var/lib/qpidd/ |25 MB | Not Applicable
+|/var/lib/pulp |1 MB |300 GB
 endif::[]
+|/var/lib/pgsql |100 MB |10 GB
+|/usr |3 GB |Not Applicable
+|/opt/puppetlabs |500 MB |Not Applicable
 |====
 
+ifdef::satellite[]
 == [[storage-el-7]]{EL} 7
 
-.Storage Requirements for a {ProjectServer} Installation
+.Storage Requirements for {SmartProxyServer} Installation
 [cols="1,1,1",options="header"]
 |====
 |Directory |Installation Size |Runtime Size
-
-|/var/log/ |10 MB |10 GB
-
-|/var/opt/rh/rh-postgresql12 |100 MB |20 GB
-
-|/usr | 3 GB | Not Applicable
-
-|/opt | 3 GB | Not Applicable
-
-|/opt/puppetlabs | 500 MB | Not Applicable
-
-ifdef::katello,satellite,orcharhino[]
-|/var/lib/pulp/ |1 MB |300 GB
-
-|/var/lib/qpidd/ |25 MB | Not Applicable
-endif::[]
-|====
-endif::[]
-
-ifdef::foreman-deb[]
```

Comment:
The Debian table must stay.

Reply:
Blarg, hard to keep all the if-else straight sometimes with adoc. Should be back now.